lib/tasks/webpack.rake

```diff
@@ -14,6 +14,6 @@ namespace :webpack do
       raise "Can't find our webpack config file at #{config_file}"
     end
 
-    sh "#{webpack_bin} --bail --config #{config_file}"
+    sh "#{webpack_bin} --config #{config_file}" --bail
```

Comment:
I .. hm. How does this fix it? Woudln't moving the --bail outside of the quotes be a syntax error?